src/GoogleFit/FitOnFhir.GoogleFit.Tests/QueueServiceTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Azure;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+using Azure.Storage.Sas;
+using FitOnFhir.Common.Interfaces;
+using FitOnFhir.Common.Services;
+using FitOnFhir.Common.Tests.Mocks;
+using FitOnFhir.GoogleFit.Common;
+using FitOnFhir.GoogleFit.Tests.Mocks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+using NSubstitute.ReturnsExtensions;
+using Xunit;
+using QueueMessage = FitOnFhir.Common.Models.QueueMessage;
```

Comment:
A few of these using statements are not needed, which causes my build to fail.

Reply:
I've noticed that whatever it is that tries to catch these, is not consistent.  Oftentimes I can build just fine, and then on subsequent builds, it will notice there are unused using statements and report a compile error.